packages/client/src/index.test.ts

```diff
@@ -58,7 +53,11 @@ describe('init client', () => {
       expect(1).toEqual(1);
     }
 
-    const tokenset = await client.handleLoginCallback(process.env.CODE_VERIFIER, process.env.CODE);
+    const tokenset = await client.handleLoginCallback(
+      'http://localhost:3000/callback',
+      process.env.CODE_VERIFIER || '',
+      process.env.CODE || ''
+    );
```

Comment:
why do we need envs when we have `codeVerifier` generated? UTs shouldn't involve envs IMO.

Reply:
this redirect needs a browser (login to logto.dev), I do this by 2 steps (1. get redirect url and verifier 2. manually copy & paste to browser and get a login code, then set them to envs and test `handleLoginCallback` will change to `cypress` or sth soon.

Reply:
as discussed offline, will remove e2e part in UTs